sceptre/cli/template.py

```diff
@@ -137,3 +137,32 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="stack-name", short_help="Reveal the stack name or names.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def stack_name_command(ctx, path):
+    """
+    Show the stack names of all stacks.
+    \f
+
+    :param path: The path to execute the command on.
+    :type path: str
+    """
+    context = SceptreContext(
+        command_path=path,
+        project_path=ctx.obj.get("project_path"),
+        user_variables=ctx.obj.get("user_variables"),
+        options=ctx.obj.get("options"),
+        ignore_dependencies=ctx.obj.get("ignore_dependencies")
+    )
+
+    plan = SceptrePlan(context)
+    responses = plan.stack_name()
+
+    chop_output = len(responses) == 1
+    for response in responses.values():
+        output = response.split(" ")[0] if chop_output else response
```

Comment:
What are the values going to look like here? I'm a bit confused about why you'd need to split the values by spaces.